dataikuapi/dss/project.py

```diff
@@ -865,6 +866,36 @@ def create_model_comparison(self, name, prediction_type):
         mec_id = res['id']
         return DSSModelComparison(self.client, self.project_key, mec_id)
 
+    def get_from_full_id(self, full_id):
```

Comment:
A use case is to retrieve a ME, a lab model, or a SMV by its full id. It is useful in the context of MES/MC. As it seems useful elsewhere, we put it there.
Regarding the naming, we hesitated a lot, but this is not a model, not an evaluation...
It seems to me that the docstring is clear and that it can be further extented to other id types, thus the very generic naming.

Reply:
So, after discussion, we should:
- name it `get_evaluation_like_from_full_id` ;
- relocate it in the model comparison / model evaluation store scope.

Reply:
So this function should be used like this:

```
my_model_comparison.get_evaluation_like_from_full_id(my_full_id)
```